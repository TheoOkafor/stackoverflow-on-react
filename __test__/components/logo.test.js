import React from 'react';
import Logo from '../../src/components/header/Logo';
import renderer from 'react-test-renderer';

test('Test that the Logo displays', () => {
  const component = renderer.create(
    <Logo />
  );
  expect(component).toMatchSnapshot();
});
