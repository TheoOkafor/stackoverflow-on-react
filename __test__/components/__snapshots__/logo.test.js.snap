// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test that the Logo displays 1`] = `
<div
  className="logo"
>
  <a
    href=""
  >
    <p
      className="home"
    >
      StackOverflow
      <small>
        -lite
      </small>
    </p>
  </a>
</div>
`;
